vagrant/provision/once.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+#== Import script args ==
+
+github_token=$(echo "$1")
+timezone=$(echo "$2")
+
+#== Bash helpers ==
+
+function info {
+  echo " "
+  echo "--> $1"
+  echo " "
+}
+
+#== Provision script ==
+
+info "Allocate swap for MySQL 5.6"
+fallocate -l 2048M /swapfile
+chmod 600 /swapfile
+mkswap /swapfile
+swapon /swapfile
+echo '/swapfile none swap defaults 0 0' >> /etc/fstab
+
+info "Configure locales"
+sudo update-locale LC_ALL="C"
+sudo dpkg-reconfigure locales
+
+info "Configure timezone"
+echo ${timezone} | sudo tee /etc/timezone
+sudo dpkg-reconfigure --frontend noninteractive tzdata
+
+info "Prepare root password for MySQL"
+debconf-set-selections <<< "mysql-server-5.6 mysql-server/root_password password \"''\""
+debconf-set-selections <<< "mysql-server-5.6 mysql-server/root_password_again password \"''\""
+echo "Done!"
+
+info "Update OS software"
+sudo apt-get update
+sudo apt-get upgrade -y
+
+info "Install additional software"
+sudo apt-get install -y git php5-cli php5-intl php5-mysqlnd php5-gd php5-fpm nginx mysql-server-5.6
```

Comment:
I think we should test current realization. And if it OK, merge it. After that it'll be comfortable to add new functionality (we'll open other issues/PRs for them).